new/PRS14298ZaraAbreham.xml

```diff
@@ -0,0 +1,59 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS14298ZaraAbreham" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Zarʾa ʾAbrǝhām</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2024-01-23">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person>
+                    <persName xml:lang="gez" xml:id="n1">ዘርአ፡ አብርሃም፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Zarʾa ʾAbrǝhām</persName>
+                    <floruit notBefore="1480" notAfter="1550">End of 15th century or first half of 16th century</floruit>
+                </person>
+                <listRelation>
+                    <relation name="snap:FatherOf" active="PRS14298ZaraAbreham" passive="PRS14299Faraday"/>
```

Comment:
maybe it could be noted that a spiritual relationship is likely? (if that is the case)

Reply:
I think the relationship is spiritual in which case `snap `is not appropriate

Reply:
Do you think it is appropriate to transform it to a dc:relation or would you suggest do omit this information.
E.g.
```
<relation name="dc:relation" active="PRS14298ZaraAbreham" passive="PRS14299Faraday"><desc>He is called the "father"
                        of <persName ref="PRS14299Faraday"/>, what is probably a spiritual relationship.</desc></relation>
```

Reply:
I think the dc:relation is fine